transport/tchannel/channel_outbound_test.go

```diff
@@ -155,95 +155,138 @@ func TestChannelOutboundHeaders(t *testing.T) {
 }
 
 func TestChannelCallSuccess(t *testing.T) {
-	server := testutils.NewServer(t, nil)
-	defer server.Close()
-	serverHostPort := server.PeerInfo().HostPort
-
-	server.GetSubChannel("service").SetHandler(tchannel.HandlerFunc(
-		func(ctx context.Context, call *tchannel.InboundCall) {
-			assert.Equal(t, "caller", call.CallerName())
-			assert.Equal(t, "service", call.ServiceName())
-			assert.Equal(t, tchannel.Raw, call.Format())
-			assert.Equal(t, "hello", call.MethodString())
+	tests := []struct {
+		msg                   string
+		withServiceRespHeader bool
+	}{
+		{
+			msg: "test channel call success with service name response header with response service name header",
```

Comment:
Let's shorten these messages to a few words.

Consider some change broke this test and this error message showed up. It's very verbose. I recommend (for your learning) intentionally making this test fail to see the error that will show up.

Reply:
Resolved.